scripts/release.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
[q] Would `-x` be useful as well for debugging or we don't care?

Reply:
if I add it you'll see the script running
```bash
./scripts/release.sh 123
+ VERSION=123
+ [[ -z 123 ]]
+ read -p 'Are you sure to release v123? ' -n 1 -r
Are you sure to release v123?
```
a bit ugly IMO

Reply:
Yeah that's fair!